frontend/config/environment.js

```diff
@@ -63,10 +64,10 @@ module.exports = function(environment) {
   };
 
   if (environment === 'development') {
-    ENV['ember-simple-auth-token'].refreshAccessTokens = false;
+    ENV.host = 'http://localhost:3000';
```

Comment:
I think this becomes meaningless noop if `ember-cli-mirage.enabled` = true, but I would like matt to weigh in. Can update to be conditional if necessary, or if more clear.

Reply:
When mirage is enabled, it intercepts all network requests, so the host is ignored.